plugin_tests/client/minervaSpec.js

```diff
@@ -42,7 +42,7 @@ describe('Main view', function () {
                     var existingCreateWidget = layer.createWidget;
                     layer.createWidget = function (widgetName) {
                         if (widgetName === 'slider') {
-                            return {};
+                            return { canvas: function () { } };
```

Comment:
At least on the latest GeoJS, I don't think we need this `beforeEach` and `afterEach`.  At least, locally, the tests all pass without these.